src/engraving/style/style.cpp

```diff
@@ -420,7 +420,7 @@ void MStyle::read(XmlReader& e, compat::ReadChordListHook* readChordListHook)
             }
         } else if (tag == "measureNumberOffset" && m_version < 440) { // pre-4.4 typo
             set(Sid::measureNumberPosAbove, PointF(e.readPoint()));
-        } else if (tag == "measureNumberPosAbove" && m_version < 440) { // pre-4.4 typo
+        } else if (tag == "measureNumberPosAbove" && m_version <= 440) { // pre-4.4 typo
             set(Sid::mmRestRangePosAbove, PointF(e.readPoint()));
```

Comment:
Probably should be `<` rather than `<=` and then the typo fixed in the MSCX test files.

Reply:
I'll try that next, now with most other tests being fixed and out of the way. Bit of a problem: many vtest scores are mscz files.

Reply:
Done, weren't that many after all